reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -99,7 +100,7 @@
  * {@link #subscribe(Subscriber)} used internally for {@link Context} passing. User
  * provided {@link Subscriber} may
  * be passed to this "subscribe" extension but will loose the available
- * per-subscribe @link Hooks#onLastOperator}.
+ * per-subscribe @link Hooks#onSubscribe}.
```

Comment:
I think this should point to `Hooks#onLastOperator(Function)` (the public setter)